pkg/operator/cluster/ceph/mon/spec.go

```diff
@@ -146,14 +156,63 @@ func (c *Cluster) monContainer(config *monConfig, fsid string) v1.Container {
 			k8sutil.ConfigOverrideMount(),
 		},
 		Env: []v1.EnvVar{
+			PodNameEnvVar(),
+			k8sutil.PodIPEnvVar(k8sutil.PublicIPEnvVar),
 			k8sutil.PodIPEnvVar(k8sutil.PrivateIPEnvVar),
-			PublicIPEnvVar(config.PublicIP),
 			ClusterNameEnvVar(c.Namespace),
-			EndpointEnvVar(),
+			EndpointsEnvVar(),
 			SecretEnvVar(),
 			AdminSecretEnvVar(),
 			k8sutil.ConfigOverrideEnvVar(),
 		},
 		Resources: c.resources,
+		/*ReadinessProbe: &v1.Probe{
```

Comment:
ohh you are using the readiness probes here. Nice.
Why are they commented out?

Reply:
Can we get a liveness/readyness probe that will call an executable and run a ceph command to check whether if the monitor is in quorum?

Reply:
I uncommented it now, but I have to test it again to make sure the probes work correctly.
Command probes to check quorum wouldn't be too good, because if the quorum is lost "all" mons would fail the probe and maybe begin restarting.
An "external" check seems better from my perspective.